Gorilla/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+	"text/template"
+
+	"gorm.io/gorm"
+
+	"github.com/gorilla/mux"
+	_ "github.com/lib/pq"
+)
+
+var (
+	postgresDB *gorm.DB
+	tpl        *template.Template // Declare tpl here for global access
+)
+
+func main() {
+	var err error
+
+	/*---------------------
+	* Load env vars
```

Comment:
Technical debt?  I think it is fine to hardcode like this but would be nice to load the DATABASE_URL from the .env.local in the root.

Reply:
Alright! I added it so it now loads env-vars in from the root :))